tests/sentry/api/endpoints/test_project_transfer.py

```diff
@@ -99,3 +105,95 @@ def test_rate_limit(self):
             response.content
             == b'"You are attempting to use this endpoint too frequently. Limit is 3 requests in 3600 seconds"'
         )
+
+    def test_transfer_project_to_current_organization(self):
+        owner_a = self.create_user("a@example.com")
+        self.create_useremail(owner_a, "shared@example.com")
+        org_a = self.create_organization(name="Org A", slug="org-a", owner=owner_a)
+
+        project = self.create_project(organization=org_a)
+        self.login_as(user=owner_a)
+
+        url = reverse(
+            "sentry-api-0-project-transfer",
+            kwargs={"organization_id_or_slug": org_a.slug, "project_id_or_slug": project.slug},
```

Comment:
Does this call mean "transfer `project` (which belong to `org_a`) to `org_a`"?

Reply:
When transferring, you can't choose which organization gets the project, only an email and the recipient decides the organization